invisible_cities/reco/paolina_functions.py

```diff
@@ -289,18 +289,21 @@ def make_tracks(evt_number       : float,
     return tc
 
 
-def drop_end_point_voxels(voxels: Sequence[Voxel], energy_threshold: float, min_vxls: int = 3) -> Sequence[Voxel]:
+def drop_end_point_voxels(voxels: Sequence[Voxel],
+                          energy_threshold: float,
+                          min_vxls: int = 3,
+                          contiguity: Contiguity = Contiguity.CORNER) -> Sequence[Voxel]:
```

Comment:
We also tend to align things like this:
```suggestion
def drop_end_point_voxels(voxels           : Sequence[Voxel],
                          energy_threshold : float,
                          min_vxls         : int = 3,
                          contiguity       : Contiguity = Contiguity.CORNER) -> Sequence[Voxel]:
```